src/pages/trader-tools/_style.js

```diff
@@ -184,6 +184,23 @@ export const CalculatorOutputSymbol = styled.label`
 
 export const CalculatorBody = styled.div`
     padding: 2.4rem;
+    /* stylelint-disable property-no-vendor-prefix */
+    ul::-webkit-scrollbar {
```

Comment:
in this case, lets make our own scrollbar components in the future with this code